lib/utils.js

```diff
@@ -22,4 +26,20 @@ var buildPolicyString = function(policyObject) {
   }, '').trim();
 };
 
-module.exports = { buildPolicyString: buildPolicyString };
+/**
+ * Returns the configuration stored in `config/content-security-policy.js`.
+ * Returns an empty object if that file does not exist.
+ *
+ * @param {string} projectRoot
+ * @return {object}
+ */
+const readConfig = function(projectRoot, environment) {
+  let configPath = path.join(projectRoot, 'config/content-security-policy.js');
```

Comment:
This logic needs to be tweaked a bit, the `config` folder isn't always in a static location.

[Here is a similar utility](https://github.com/emberjs/ember-optional-features/blob/410c8dc0e2273e4d0f538c0e233a9ec659c5e291/utils.js#L3-L11) I built a little while back, but the tldr is _basically_:

```js
function getConfigPath(projectPkg, projectRoot) {
  let configDir = 'config';

  if (projectPkg['ember-addon'] && projectPkg['ember-addon']['configPath']) {
    configDir = projectPkg['ember-addon']['configPath'];
  }

  return path.join(projectRoot, configDir, 'content-security-policy.js');
}
```

For addon's that depend on ember-cli-content-security-policy the config path would then be `tests/dummy/config/content-security-policy.js` (which allows them to use this addon for their deployed demo app for example), other projects need to use the `config` folder for "something else" (don't ask :weary:) and therefore customize `ember-addon.configPath` to be `ember-cli-config`.